scripts/automation/updateManagedAppleId.ps1

```diff
@@ -14,6 +14,9 @@ if (-not (Get-InstalledModule -Name JumpCloud)) {
     Write-Host "Installing JumpCloud PowerShell Module"
     Install-Module JumpCloud -Force
 }
+if ((Get-InstalledModule -Name JumpCloud.SDK.V1).Version -notmatch 0.0.27) {
```

Comment:
If version is > than .27 this will still trigger this condition. Let's review this part